--- openjpeg_v1_4_sources_r697/configure.ac.orig	2011-01-02 19:14:55.000000000 +0100
+++ openjpeg_v1_4_sources_r697/configure.ac	2012-01-21 11:33:38.113810775 +0100
@@ -64,7 +64,8 @@
 LIB_Z=""
 AC_CHECK_LIB(z,inflate,LIB_Z="-lz",,)
 #
-LIBS="$LIBS $LIB_MATH $LIB_Z"
+AC_SUBST([LIB_MATH])
+AC_SUBST([LIB_Z])
 #
 AC_PATH_PROG([PKGCONFIG], [pkg-config])
 #
@@ -136,7 +137,6 @@
     if test -n "$pngincludes" ; then
       with_libpng="yes"
       png_header_found="yes"
-      LIBS="$LIBS $pnglibs"
       AC_DEFINE(HAVE_LIBPNG, [1], [define to 1 if you have libpng])
     fi
   fi
@@ -160,13 +160,6 @@
 test "$enable_tiff" = "no"  && with_libtiff="no")
 #
 if test "x$with_libtiff" = xyes ; then
-#TIFF compiled with JPEG and JBIG support?
- libjpeg=""
- AC_CHECK_LIB(jpeg,jpeg_read_header,libjpeg=-ljpeg,,)
- libjbig=""
- AC_CHECK_LIB(jbig,jbg_enc_init,libjbig=-ljbig,,)
- libjbig85=""
- AC_CHECK_LIB(jbig85,jbg85_enc_init,libjbig85=-ljbig85,,)
 #standard path
  with_libtiff="no"
  savedLIBS="$LIBS"
@@ -177,7 +170,6 @@
   AC_CHECK_HEADER(tiff.h,tiff_header_found="yes",,)
   if test "$tiff_header_found" = "yes" ; then 
    with_libtiff="yes"
-   LIBS="$LIBS -ltiff $libjpeg $libjbig $libjbig85"
    AC_DEFINE(HAVE_LIBTIFF, [1], [define to 1 if you have libtiff])
   fi
  fi
@@ -211,7 +203,7 @@
     AC_DEFINE(HAVE_LIBTIFF, [1], [define to 1 if you have libtiff])
    fi
   fi
-  LIBS="$savedLIBS $tifflibs $libjpeg $libjbig $libjbig85"
+  LIBS="$savedLIBS"
  fi
 fi
 #
@@ -295,7 +287,7 @@
     lcms2includes=""
     lcms2libs=""
   fi
-  LIBS="$savedLIBS $lcms2libs"
+  LIBS="$savedLIBS"
 fi
 #
 AC_MSG_RESULT(lcms2 is usable:$with_liblcms2)
@@ -369,7 +361,7 @@
      lcms1includes=""
      lcms1libs=""
    fi
-   LIBS="$savedLIBS $lcms1libs"
+   LIBS="$savedLIBS"
  fi
 AC_MSG_RESULT(lcms is usable:$with_liblcms1)
 fi
--- openjpeg_v1_4_sources_r697/libopenjpeg/Makefile.am.orig	2012-01-21 11:34:33.197145953 +0100
+++ openjpeg_v1_4_sources_r697/libopenjpeg/Makefile.am	2012-01-21 11:43:16.683830149 +0100
@@ -7,6 +7,7 @@
 
 AM_CPPFLAGS = -I.. -I.
 
+libopenjpeg_la_LIBADD = $(LIB_MATH)
 libopenjpeg_la_LDFLAGS = -no-undefined \
   -version-info $(CURRENT):$(BUILD):$(MINOR)
 
--- openjpeg_v1_4_sources_r697/jpwl/Makefile.am.orig	2012-01-21 11:45:58.817168910 +0100
+++ openjpeg_v1_4_sources_r697/jpwl/Makefile.am	2012-01-21 11:46:27.237169861 +0100
@@ -5,6 +5,7 @@
 
 lib_LTLIBRARIES = libopenjpeg_JPWL.la
 
+libopenjpeg_JPWL_la_LIBADD = $(LIB_MATH)
 libopenjpeg_JPWL_la_LDFLAGS = -no-undefined \
   -version-info $(CURRENT):$(BUILD):$(MINOR)
 
